### PR TITLE
Create IssuesFAQ.md

### PR DESCRIPTION
**Pull request by blackcrack**
~~For hold out bad persons~~ and let show the right way ;)

### CODE REVIEW

**Comment by Ibuprophen** [comment]
A Very good start but, this is not the direction that I was referring to as to where I was going to place this type of information. 

If I use some/most of this, it'll definitely require a bit of tweaking at first with the majority vote from the other developers. 


~Ibuprophen

**Comment by blackcrack** [comment]
Hi @Ibuprophen change it like you meant and make it in the right direction.. :)

**Comment by Ibuprophen** [comment]
What do you think @coddec, @ge0rdi and/or @XenHat? 

Just looking for a 2nd+ Opinion... 

~Ibuprophen

File: IssuesFAQ.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Read before you Post...
+This FaQ !
+
+§1 be peaceful, frendful and work together !
+
+§2 Mandatory informations if you have a Bug or a Error<br>
+  &nbsp;&nbsp;&nbsp;In the first Post !
+
+   * Ordering System
```
Comment (ge0rdi) [inline IssuesFAQ.md]: I guess you meant "Operating system".

File: IssuesFAQ.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+   * Architecture (x86 or x86_64)
+   * Version of the Program
```
Comment (ge0rdi) [inline IssuesFAQ.md]: "Version of Open-Shell" ?

File: IssuesFAQ.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+   * what was done, how was it done, how should it behave and how it is now.
+   * the behaver of the bug
```
Comment (ge0rdi) [inline IssuesFAQ.md]: behavior?

File: IssuesFAQ.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+§3 Or, if you invent, explain it for able to fellow and building in.<br>
```
Comment (ge0rdi) [inline IssuesFAQ.md]: Don't quite understand meaning of this ...

File: IssuesFAQ.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+<br>
+§4 Do not Bullying or Trolling this Program or Users !<br>
```
Comment (ge0rdi) [inline IssuesFAQ.md]: This whole paragraph and the rest of the text should be dropped.

File: IssuesFAQ.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+Read before you Post...
```
Comment (ge0rdi) [inline IssuesFAQ.md]: These two lines are just unnecessary.
Obviously if one got to this page, his intent is to read it.

**Comment by blackcrack** [comment]
@ge0rdi 
then change it like you need.. and if you want be more softy, do it ..
we need a tool in the hand where we can pot it in the yes and can close because "§ XY"
and have a reason why we doe something.. or remove it and delete it, 
tear it and kick it in the trash. .
 so.. you the Programmer .. \*Turn me away and walk away*

\*i throw the penny behind me*

**Comment by ge0rdi** [comment]
@blackcrack 
I'm sorry if I offended you somehow. That wasn't my intent.
I was asked for opinion on proposed changes, so I tried to provide a feedback.

Moreover I've realized that this kind of stuff belongs to Wiki page, not to repo (as it is unrelated to source code).

I'd suggest to close this PR.
We can eventually create Wiki page with description about how to create issues.

**Comment by coddec** [comment]
I second, to put this information to WiKi. 👍
@ge0rdi @Ibuprophen

**Comment by coddec** [comment]
Thanks for your input @blackcrack , @ge0rdi  was just trying to propose fixes like fixing bugs, or call it collaborate ;)
Relax and again thanks for your input :)

**Comment by blackcrack** [comment]
@ge0rdi => ... *g*

we neeeed a tool for some situation for say, "sorry dude, you to agro or to angry, take a outtime and try it again, you know, there it is written and this be us conditions for cooperate, this what you do, is not correct " in short Wiki §YX in cooperation .. point..
@coddec, ehh where is the wiki ??

best regards
Blacky

**Comment by coddec** [comment]
> @ge0rdi => ... *g*
> 
> we neeeed a tool for some situation for say, "sorry dude, you too agro or too angry, take a outtime and try it again, you know, there it is written and this be us conditions for cooperate, this what you do, is not correct " in short Wiki §YX in cooperation .. point..
> @coddec, ehh where is the wiki ??
> 
> best regards
> Blacky

The link for wiki :)
https://github.com/Open-Shell/Open-Shell-Menu/wiki

**Comment by ge0rdi** [comment]
Closing this as it doesn't belong to the repo.

**Pull request closed without merging**